packages/esl-website/views/examples/carousel/css-fade.sample.njk

```diff
@@ -17,12 +17,19 @@ tags: carousel-sample
       {% for i in range(0, 4) -%}
         <li esl-carousel-slide {{ 'active' if loop.first }}>
           <div class="card">
-            <div class="img-container img-container-16-9" esl-image-container>
+            <div class="card-bg img-container" esl-image-container>
               <img class="img-fade img-cover"
                    alt="{{ 'Carousel slide ' + loop.index }}"
-                   src="{{ '/assets/carousel/' + loop.index + '-sm.jpg' | url }}"
+                   src="{{ '/assets/carousel/' + (loop.index + 2) + '-sm.jpg' | url }}"
                    loading="lazy" />
-              <div class="h1 text-slide text-white">Slide {{ loop.index }}</div>
+            </div>
+            <div class="card-content card-content-spaced text-white text-center">
+              <div class="h1">Slide {{ loop.index }}</div>
+              {% if loop.index <= 2 %}
+                <!-- paragraph {{1 + (2 - i)}} -->
```

Comment:
do you still need it? here and below

Reply:
Yep, btw it is not a comment (well, not a plain comment I should say), it is an autoreplaceble instruction for UIP. It is visible in editor as a comment but rendered as lorem ipsum text.